### PR TITLE
presto 0.181

### PR DESCRIPTION
**Pull request by shigemk2**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Upgrade presto to 0.181

### CODE REVIEW

File: Formula/presto.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ class Presto < Formula
   resource "presto-cli" do
     version cli_version
     url "https://search.maven.org/remotecontent?filepath=com/facebook/presto/presto-cli/#{version}/presto-cli-#{version}-executable.jar"
```
Comment (fxcoudert) [inline Formula/presto.rb]: Please remove `#{version}` from the `url`, this is not allowed as per Homebrew style. Explicitly state the version in the URL.

**Review by fxcoudert — DISMISSED** [review]

File: Formula/presto.rb (hunk 1 above)
Comment (shigemk2) [inline Formula/presto.rb]: OK, sure.
But I have a question. Where is Homebrew style?
I couldn't find Homebrew's coding terms.

Comment (fxcoudert) [inline Formula/presto.rb]: Some of it is in the [formula cookbook](https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md) (unfortunately not all). Some of it is enforced by the audits. Some of it is passed down by PR review.

Comment (shigemk2) [inline Formula/presto.rb]: I got it.

>  you’ll need to explicitly add the correct version to the formula and then save the formula.

Thank you so much! I'll fix it.

**Comment by shigemk2** [comment]
@fxcoudert I fixed codes.

File: Formula/presto.rb [hunk 2]
```diff
@@ -11,8 +11,8 @@ class Presto < Formula
   cli_version = version
   resource "presto-cli" do
     version cli_version
-    url "https://search.maven.org/remotecontent?filepath=com/facebook/presto/presto-cli/#{version}/presto-cli-#{version}-executable.jar"
-    sha256 "f9ac124ce2acae655df927e860016f6db333ec112d2ba3bf7b3e9bb93b057965"
+    url "https://search.maven.org/remotecontent?filepath=com/facebook/presto/presto-cli/0.181/presto-cli-0.181-executable.jar"
```
Comment (ilovezfs) [inline Formula/presto.rb]: the version still needs to be specified here.

```
version "0.181"
```

File: Formula/presto.rb [hunk 3]
```diff
@@ -11,8 +11,8 @@ class Presto < Formula
   cli_version = version
   resource "presto-cli" do
     version cli_version
```
Comment (ilovezfs) [inline Formula/presto.rb]: this line should be deleted

File: Formula/presto.rb [hunk 4]
```diff
@@ -1,18 +1,16 @@
...
   bottle :unneeded
 
   depends_on :java => "1.8+"
 
-  cli_version = version
   resource "presto-cli" do
-    version cli_version
-    url "https://search.maven.org/remotecontent?filepath=com/facebook/presto/presto-cli/#{version}/presto-cli-#{version}-executable.jar"
-    sha256 "f9ac124ce2acae655df927e860016f6db333ec112d2ba3bf7b3e9bb93b057965"
+    url "https://search.maven.org/remotecontent?filepath=com/facebook/presto/presto-cli/0.181/presto-cli-0.181-executable.jar"
```
Comment (ilovezfs) [inline Formula/presto.rb]: Please add
```
    version "0.181"
```
under the url

File: Formula/presto.rb [hunk 5]
```diff
@@ -55,7 +54,7 @@ def install
     EOS
 
     resource("presto-cli").stage do
-      bin.install "presto-cli-#{version}-executable.jar" => "presto"
+      bin.install "presto-cli-0.181-executable.jar" => "presto"
```
Comment (ilovezfs) [inline Formula/presto.rb]: This should still use the variable

**Review by ilovezfs — DISMISSED** [review]

File: Formula/presto.rb [hunk 6]
```diff
@@ -97,6 +96,6 @@ def plist; <<-EOS.undent
 
   test do
     system bin/"presto-server", "run", "--help"
-    assert_equal "Presto CLI #{version}", shell_output("#{bin}/presto --version").chomp
+    assert_equal "Presto CLI 0.181", shell_output("#{bin}/presto --version").chomp
```
Comment (ilovezfs) [inline Formula/presto.rb]: This should still use the variable

**Comment by ilovezfs** [comment]
Thanks for your first contribution to Homebrew, @shigemk2! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**